pkg/util/utils_test.go

```diff
@@ -136,3 +139,32 @@ func TestFriendlyDuration(t *testing.T) {
 		}
 	}
 }
+
+func TestCommandExecutesSuccessfully(t *testing.T) {
+	currentDir, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Unexpected error: %v", err)
+	}
+	dummybinary := filepath.Join(currentDir, "..", "..", "test", "testdata", "dummybinary")
+
+	if runtime.GOOS == "windows" {
+		dummybinary = filepath.Join(currentDir, "..", "..", "test", "testdata", "dummybinary_windows.exe")
+	}
+	if runtime.GOOS == "darwin" {
+		dummybinary = filepath.Join(currentDir, "..", "..", "test", "testdata", "dummybinary_darwin")
+	}
+
+	testData := []struct {
+		command string
+		exists  bool
+	}{
+		{"blahh", false},
+		{dummybinary, true},
+	}
+	for _, v := range testData {
+		got := CommandExecutesSuccessfully(v.command)
+		if got != v.exists {
+			t.Errorf("Expected %v Got %v", v.exists, got)
+		}
+	}
+}
```

Comment:
This test should fail, as the `dummybinary*` is not present in the `PATH` if `exec.LookPath()` only looks for the file in `PATH`, maybe it behaves differently when absolute path is passed.

Reply:
See comment above.

Reply:
Will fix this as per this [comment](https://github.com/minishift/minishift/pull/1822/files#r158220129).

Reply:
So right, these tests are not correct.